flow/designs/rapidus2hp/hercules_is_int/rules-base.json

```diff
@@ -163,11 +158,11 @@
         "compare": "<="
     },
     "cts__timing__setup__ws": {
-        "value": -344.0,
+        "value": -390.0,
         "compare": ">="
     },
     "cts__timing__setup__tns": {
-        "value": -2480000.0,
+        "value": -3510000.0,
```

Comment:
It's curious that we had a big jump on tns during CTS, but the GRT timing metrics had good improvement. I guess the new placement makes it easier for GRT to use resistance-aware and avoid M3 routes.